t/10-tests_overview.t

```diff
@@ -151,6 +151,9 @@ $get->element_exists('#res_DVD_i586_kde',                           'job from gr
 $get->element_exists('#res_GNOME-Live_i686_RAID0 .state_cancelled', 'another job from group 1001');
 $get->element_exists('#res_NET_x86_64_kde .state_running',          'job from group 1002 is shown');
 
+$get = $t->get_ok('/tests/overview?distri=opensuse&version=13.1&groupid=1001&groupid=1002')->status_is(404);
```

Comment:
returning 404 is fine, but I hope you still redirect to some other page instead of showing the API routes